client/containers/HomePage.js

```diff
@@ -18,13 +18,12 @@ class HomePage extends Component {
   }
   render() {
     const { condition, place } = this.props;
-    console.log(Object.keys(condition).length === 1 ? "homepageClick1" : "homepageClick");
     return (
       <div className="homePageWrapper">
         <Place place={place} />
         <div className="searchWrapper">
           <Condition condition={condition} action={this.handleOnConditionChange} />
-          <Button onClick={this.handleOnClick} theme={Object.keys(condition).length === 1 ? "homepageClickGrey" : "homepageClick"} />
+          <Button onClick={this.handleOnClick} theme={Object.keys(condition).length === 1 ? 'homepageClickGrey' : 'homepageClick'} />
```

Comment:
1. condition may have other keys defined instead of just lat lng, like radius, so the button wont be grey?
2. homepageClickGrey has no style defined
3. tests please